text/0016-katello-as-a-plugin/0022-converge-katello-foreman-ports.md

```diff
@@ -0,0 +1,61 @@
+- Feature Name: Converge Katello and Foreman Ports
+- Start Date: 2016-10-11
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+The Foreman and Katello installations have a varying use of ports for common services (e.g. Capsules use 9090 and Foreman Proxy uses 8443) as well Katello adds services that require use of common ports. Based on user feedback the goals of port changes should be:
+
+ * Least number of ports
+ * Use of well known or well-defined ports for security contexts
+
+# Motivation
+[motivation]: #motivation
+
+The difference in port usage causes differences in installations making it harder to add Katello to a Foreman, have common documentation and SELinux rules. The expectation is to converge on a common port usage paradigm that can be used for current services and as a guideline when adding new services in the future.
+
+# Detailed design
+[design]: #detailed-design
+
+### Capsule vs Smart Proxy Ports
+
+The Foreman Proxy uses 8443 by default on installation while the Capsule install for Katello uses 9090. This was originally done due to Candlepin's use of 8443; however, given Candlepin is a local installation the port it uses matters less.
```

Comment:
so in all in one default install, subscription manager clients connect directly to 8443(candlepin) or rather it get proxied via katello itself?